pkgs/by-name/dp/dprint/plugins/default.nix

```diff
@@ -34,6 +35,30 @@ let
         mkdir -p $out
         cp $src $out/plugin.wasm
       '';
+      doInstallCheck = true;
+      nativeInstallCheckInputs = [
+        dprint
+      ];
+      preInstallCheck = ''
+        # dprint caches plugins before running
+        cacheDir="$NIX_BUILD_TOP/.cache"
+        mkdir "$cacheDir"
+        export DPRINT_CACHE_DIR="$cacheDir"
+
+        # Keep the working directory clean to prevent dprint from picking up unexpected files
+        tmpDir="$NIX_BUILD_TOP/.tmp"
+        mkdir "$tmpDir"
+        cd "$tmpDir"
+      '';
```

Comment:
```suggestion
```

Reply:
>add suggestion to batch

I didn't use this feature ever, is it disabled in this case?

![image](https://github.com/user-attachments/assets/614b1317-dc54-4c15-a414-fe42cdf6d824)

Reply:
It only works from the /files tab

Reply:
Thanks! Applied in https://github.com/NixOS/nixpkgs/pull/402056/commits/075e17a90a114be54f67e7efae1c829b4cd027a9